build_run_docker.sh

```diff
@@ -372,6 +380,26 @@ SINGLE_API_IP=''
 MULTI_API_IPS=''
 
 VOLUMES=()
+
+add_volume_to_template() {
+    local VOLUME_NAME
+    if [ -n "$VOLUMES_DIR" ]; then
+        VOLUME_NAME="${VOLUMES_DIR}$1"
+        mkdir -p "$VOLUME_NAME"
+        if [ -n "$3" ]; then
```

Comment:
Maybe name this variable first `IS_NDBD_DATA_DIR=$3`.
Also, perhaps add a comment why you're creating 64 dirs.. 🙃

Reply:
Yeah perhaps just do

```
VOLUME_NAME=$1
TARGET_DIRNAME=$2
IS_NDBD_DATA_DIR=$3
```

and then just overwrite `$VOLUME_NAME` for the first if statement?

Reply:
The Dockerfile does it:
`RUN for i in $(seq 64); do mkdir -p $NDBD_DATA_DIR/$i; done`
And when I mount a local directory, it is created empty and ndbd complains about not finding a subdirectory named after its nodeId.
So far I think I understand it, but why isn't this a problem when using docker volumes? You have any idea?

Reply:
Pushed but with TODO